docs/setup_vscode_wsl.md

```diff
@@ -219,69 +193,63 @@ In EECS 280, you'll do this to any file that ends in `.starter`.
 | <img src="images/vscode_wsl_027.png" height="512px" /> | <img src="images/vscode_wsl_028.png" height="512px" /> |
 
 <div class="primer-spec-callout info" markdown="1">
-**Pro-tip:** You can also rename files the command line, for example:
+**Pro-tip:** You can also rename files from the command line, for example:
 ```console
 $ mv stats_tests.cpp.starter stats_tests.cpp
 ```
 </div>
 
-## Compile and Run
-VS Code uses an executable you build at the command line.
+### Add new files
 
-First, compile and run your executable at the command line.
-```console
-$ touch stats.cpp  # Needed for EECS 280 P1
-$ make main.exe
```

Comment:
It seems like at this point in the tutorial, a person following along without P1 starter files might get lost.  It would be nice if some version of "hello world" could work here.

It's tricky to thread the needle and make this directly applicable to P1 while also making it mostly make sense in a stand-alone (no starter files) context, but that's my hope.  Do you have any ideas?

Reply:
Could you elaborate on where they might get lost? Would it help to add a mention of the sample `main.cpp` and `Makefile` earlier (currently in the debugging section?). Something like, "if you're not in EECS 280 and you need a set of starter files, you can use these".

But, as a general principle, I think we should prioritize students in EECS 280.

Maybe we could make a separate issue to review the tutorials through the lens of someone not in 280?

Reply:
@jamesjuett and I talked about this in person and decide to go with

> add a mention of the sample main.cpp and Makefile earlier (currently in the debugging section?).

Reply:
Drew and I discussed briefly in-person, and we determined lifting the sample `main.cpp` and `Makefile` higher in the tutorial is a good plan for this PR, with potential further improvements in the future. I'll get this done ASAP.